packages/metrics/index.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type React from "react";
+import { getInjectionToken } from "@ogre-tools/injectable";
+
+export type ClusterOverviewUIBlock = {
+  id: string;
+  Component: React.ElementType;
+  orderNumber: number;
+};
+
+export const clusterOverviewUIBlockInjectionToken = getInjectionToken<ClusterOverviewUIBlock>({
+  id: "cluster-overview-ui-block-injection-token",
+});
```

Comment:
To me, it seems that these do not belong here. Maybe add a comment to remind that these should be moved under `cluster-overview` feature whenever it emerges.

Reply:
added a comment here